### PR TITLE
Add api route for messages per Chat room

### PR DESCRIPTION
**Pull request by kanevk**
@svetlio2 @SteinPowaaa  review it, please.

### CODE REVIEW

File: app/controllers/api/v1/messages_controller.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 module Api
   module V1
     class MessagesController < ApiController
+      def index
+        # TODO: Add Pagination and order filter
+        return render status: 404, nothing: true unless chat_room
+
+        messages = Message.where(chat_room: chat_room).order(created_at: :asc)
```
Comment (scdekov) [inline app/controllers/api/v1/messages_controller.rb:8]: I think that correct order here is descending, but it doesn't matter so much unless we add pagination.

**Comment by scdekov** [comment]
:ok_hand:

File: app/controllers/api/v1/messages_controller.rb (hunk 1 above)
Comment (kanevk) [inline app/controllers/api/v1/messages_controller.rb:8]: Why descending?? As now the last message will be at the top.

Comment (scdekov) [inline app/controllers/api/v1/messages_controller.rb:8]: For now it doesn't matter, but if we have paging, we need to retrieve newest messages first, and that means desc will be correct. In the UI results will be reversed before displaying. Do you agree?

Comment (kanevk) [inline app/controllers/api/v1/messages_controller.rb:8]: We will think about it later.

**Pull request merged**